app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: Car) -> float:
+        income = 0.0
+        for i in range(len(cars)):
+            if cars[i].clean_mark <= self.clean_power:
+                income += self.calculate_washing_price(cars[i])
+                self.wash_single_car(cars[i])
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        clean_diff = self.clean_power - car.clean_mark
+        clean_eff = car.comfort_class * clean_diff * self.average_rating
+        inc = clean_eff / self.distance_from_city_center
```

Comment:
There is a potential division by zero error if `distance_from_city_center` is zero. Consider adding a check to handle this case appropriately.